assets/js/utils/renderSearchPage.js

```diff
@@ -12,7 +12,7 @@ import searchResultsTemplate from '../templates/search/searchResultsTemplate';
 import totalResults from '../templates/search/totalResultsTemplate';
 import clickTracking from '../click-tracking';
 
-export default function renderSearchPage(searchResults, urlParams, numberOfResults) {
+export default function renderSearchPage(searchResults, urlParams, numberOfResults, audience = null) {
```

Comment:
You should be able to do this without the default parameter - in javascript, by default, parameters that aren't specified will be `undefined`

```suggestion
export default function renderSearchPage(searchResults, urlParams, numberOfResults, audience) {
```